### ISSUE TITLE
Controls present under "New Collection" blade is not visible at 400% resize mode.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1027]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1216204

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre Requisite:
Screen Resolution: 1280X1024 and resize the page up to 400% zoom.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Add Collection" Menu item and select it.
Navigate through all the controls under add Collection/New Collection blade.
Resize the web page up to 400% mode.
Verify the issue.

Actual Result:
Controls present under "New Collection" blade is not visible at 400% resize mode.

Expected Result:
Controls present under "New Collection" blade should be not visible properly and clearly at 400% resize mode.

Note:
Similar issue exists under 'New Database' blade.
This issue also exists for similar controls present under 'New Graph' blade and 'New Vertex' blade of Data explorer screen for Created cosmos DB  after selecting "Gremlin(graph)" option.
Similar issue exists throughout the application.

![image](https://user-images.githubusercontent.com/8519407/132720791-94a8ae5f-f1e6-4816-ac50-e8e5e8ff591e.png)

![image](https://user-images.githubusercontent.com/8519407/132720805-8cf4829d-d2c3-4da4-bd90-7da6f9f990d2.png)
